sunpy/map/tests/test_header_helper.py

```diff
@@ -90,16 +88,19 @@ def test_make_fits_header(map_data, hpc_test_header, hgc_test_header,
     assert header['crpix1'] == 5.5
     assert header['ctype1'] == 'HPLN-TAN'
     assert u.allclose(header['dsun_obs'], hpc_test_header.frame.observer.radius.to_value(u.m))
+    assert header.get('rsun_obs') is not None
```

Comment:
I don't know how to work out this value without replicating the code in the header function.

Reply:
Well, yeah, you're basically going to have to replicate the call to `sun._angular_radius()`.  I'd do:
```suggestion
    assert u.allclose(header['rsun_ref'] * u.m, hpc_test_header.frame.rsun)
    assert u.allclose(header['rsun_obs'] * u.arcsec,
                      sun._angular_radius(header['rsun_ref'] * u.m, header['dsun_obs'] * u.m))
```